lib/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -531,6 +603,52 @@ public static function get_user_data() {
 		return static::$user;
 	}
 
+	public static function get_site_data() {
```

Comment:
`get_site_data` and `get_site_data_from_wp_global_styles` need the corresponding PHPDoc.

Reply:
:+1: [Added](https://github.com/WordPress/gutenberg/pull/50102/commits/ce3bb54eb2a4358659e7e341d86ecaea24d7b28f)!